src/script/lua_api/l_area_store.cpp

```diff
@@ -158,19 +158,7 @@ int LuaAreaStore::l_insert_area(lua_State *L)
 	LuaAreaStore *o = checkobject(L, 1);
 	AreaStore *ast = o->as;
 
-	Area a;
-
-	a.minedge = check_v3s16(L, 2);
-	a.maxedge = check_v3s16(L, 3);
-
-	a.extremifyEdges();
```

Comment:
Why have you removed this? Now, lua has to do it itself, this has to be reverted simply because it breaks the lua API. Also, it removes a feature for no real reason.